packages/libvips/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://libvips.github.io/libvips/
+TERMUX_PKG_DESCRIPTION="A fast image processing library with low memory needs"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="Thibault Meyer <meyer.thibault@gmail.com>"
+TERMUX_PKG_VERSION=8.10.5
+TERMUX_PKG_SRCURL=https://github.com/libvips/libvips/releases/download/v8.10.5/vips-8.10.5.tar.gz
```

Comment:
Don't hardcode version in URL, use `${TERMUX_PKG_VERSION}` instead.

Reply:
Done